index.d.ts

```diff
@@ -1,29 +1,29 @@
 import * as React from 'react';
 
 type Function = () => void;
+type UpdateFunction = (newEnd?: number) => void;
```

Comment:
Maybe we should map types to the original fn?
https://github.com/inorganik/countUp.js/blob/master/src/countUp.ts#L147